python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -53,6 +54,21 @@ class ComputeExposureSummaryStatsConfig(pexConfig.Config):
         doc="Mask planes that, if set, the associated pixel should not be included sky noise calculation.",
         default=("NO_DATA", "SUSPECT"),
     )
+    starSelection = pexConfig.Field(
+        doc="Field to select stars.",
+        dtype=str,
+        default='calib_psf_used'
+    )
+    starShape = pexConfig.Field(
+        doc="Name of star shape",
+        dtype=str,
+        default='base_SdssShape'
+    )
+    psfShape = pexConfig.Field(
+        doc="Name of psf shape",
+        dtype=str,
+        default='base_SdssShape_psf'
+    )
```

Comment:
...because I literally cut-and-pasted from the prior art which had the mix.  And while I have a lot of code style opinions, strangely single-or-double spaces is not something I notice.  Anyway, I'll fix to double quotes.